test/room_test.rb

```diff
@@ -9,16 +9,24 @@ def test_it_exists
   end
 
   def test_it_has_a_category
-    skip
+    # skip
     room = Room.new(:bedroom, 10, '13')
     assert_equal :bedroom, room.category
   end
 
   def test_it_can_get_area
-    skip
+    # skip
     room1 = Room.new(:bedroom, 10, '13')
     room2 = Room.new(:living_room, 15, '12')
     assert_equal 130, room1.area
     assert_equal 180, room2.area
   end
+
+  def test_is_painted?
+    room = Room.new(:bedroom, 10, "13")
+    room2 = Room.new(:living_room, 15, '12')
+    assert_equal false, room.test_is_painted?
```

Comment:
This line makes me wonder if you are not running your tests as often as you should.  This test fails with an undefined method 'test_is_painted?` which I believe is a typo.